Gruntfile.js

```diff
@@ -186,6 +186,7 @@ module.exports = function(grunt) {
         files: {
           // public api files
           'api/build/public/login/script.js': 'api/build/public/login/script.js',
+          'api/build/public/login/bowser.js': 'api/build/public/login/bowser.js',
```

Comment:
typo in `bowser`

Reply:
this script only contains the library [bowser](https://github.com/lancedikson/bowser), it's not a typo

Reply:
oh wow that's just super confusing :)))

Reply:
what file name would you expect instead?  
would adding `vendors` before/after `bowser` make it clearer maybe?

Reply:
maybe ... `lib-bowser` or something?